src/components/SEO/index.tsx

```diff
@@ -0,0 +1,98 @@
+import React from 'react'
```

Comment:
should we start splitting blog, common, docs, etc? most likely SEO will be different across those .. and number of components is getting out or hands as well

Reply:
It depends on what you want. Could you describe it more precisely and we will decide then?
Now SEO data should be same as it was before.

Reply:
we have Documentation already for components related to `/doc` - that where I would start. Move blog to Blog/, for example. keep common in the `components` root

SEO - the ways we process them is different - for blog we rely on some additional meta-fields in Mardown, for docs we don't have that, right? Thus we (or will) have different SEO components for docs/blog